docs/nidcpower/rep_caps.rst

```diff
@@ -18,10 +18,15 @@ Repeated Capabilities
 
     Repeated capabilities attributes use the indexing operator :python:`[]` to indicate the repeated capabilities.
     The parameter can be a string, list, tuple, or slice (range). Each element of those can be a string or
-    an integer. If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
-    :python:`'0:2'`
+    an integer.
 
-    Some repeated capabilities use a prefix before the number and this is optional
+    ..
+        If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
+        :python:`'0:2'`
+
+        Some repeated capabilities use a prefix before the number and this is optional.
```

Comment:
remove

Reply:
Yeah, I guess this comment is redundant with the commented example.